api-reference/10 UI Components/dxValidationSummary/3 Methods/refreshValidationGroup().md

```diff
@@ -3,7 +3,252 @@ id: dxValidationSummary.refreshValidationGroup()
+
+    //...
+
+    export class AppComponent {
+        @ViewChild('group', {static: false}) validationGroup!: DxValidationGroupComponent
+        @ViewChild('summary', {static: false}) summary!: DxValidationSummaryComponent
+        visible: boolean = true;
+        removeGroup() {
+            this.visible = false;
+        }
+        renderGroup() {
+            this.visible = true;
+            this.summary.instance.refreshValidationGroup(); // Refresh the subscription to ValidationGroup after rendering
+        }
+        validateGroup() {
+            this.validationGroup.instance.validate();
+        }
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div id="container">
+            <DxValidationGroup
+                :ref="groupRefKey"
+            >
+                <DxTextBox v-if="visible">
+                    <DxValidator>
+                        <DxRequiredRule />
+                    </DxValidator>
+                </DxTextBox> 
+                <DxTextBox v-if="visible">
+                    <DxValidator>
+                        <DxRequiredRule />
+                    </DxValidator>
+                </DxTextBox>
+                <DxValidationSummary 
+                    :ref="summaryRefKey"
+                />
+            </DxValidationGroup>
+            <DxButton
+                text="Validate"
+                @click="validateGroup"
+            />
+            <DxButton
+                text="Remove group"
+                @click="removeGroup"
+            />
+            <DxButton
+                text="Render group"
+                @click="renderGroup"
+            />
+        </div>
+    </template>
+
+    <script>
+        // ...
+        export default {
+            // ...
+            data() {
+                return {
+                    groupRefKey: 'group',
+                    summaryRefKey: 'summary',
+                    visible: true
+                }
+            },
+            methods: {
+                validateGroup() {
+                    this.validationGroup.validate();
+                },
+                removeGroup() {
+                    this.visible = false;
+                },
+                renderGroup() {
+                    this.visible = true;
+                    this.summary.refreshValidationGroup(); // Refresh the subscription to ValidationGroup after rendering
+                }
+            },
+            computed: {
+                validationGroup() {
+                    return this.$refs[this.groupRefKey].instance;
+                },
+                summary() {
+                    return this.$refs[this.summaryRefKey].instance;
+                }
+            }
+        }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    // ...
+
+    function App() {
+        const [visible, setVisible] = useState(true);
+        let validationGroup = null;
+        let summary = null;
```

Comment:
We can use `React.createRef()` here and bind this ref to a component (to not pass an init function to ref)

Reply:
Noted